File: backend/album-collection/Models/Review.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace album_collection.Models
{
    public class Review
    {
        public int Id { get; set; }
        public string Content { get; set; }
        public string Reviewername { get; set; }
        public int AlbumId { get; set; }
        public virtual Album Album { get; set; }
       

        public Review()
        {

        }
    }
}

File: frontend/src/js/Components/Review.js
export default {
    DisplayReview,
}

function DisplayReview(review){
    return `
        
        <h1><label><strong>Posted By: </strong></label>${review.reviewername}</h1>
        
        <h4><label>Review Content: </label>${review.content}</h4>

        <h4><label>Album: </label>${review.album.title}</h4>
    `;
}